news_items/011_python_autograder/index.html

```diff
@@ -52,7 +52,7 @@
 
 <img class="border rounded pt-2" src="variable_description.png" width="804px">
 
-<p> The new autograder is included by default in the <code class="user-output">grader-python</code> docker image, to take advantage of it you can edit your question's `info.json` file to include: </p> 
+<p> The new autograder is included by default in the <code class="user-output">grader-python</code> docker image, to take advantage of it you can edit your question's <code class="user-output">info.json</code> file to include: </p> 
```

Comment:
Hahaha, I keep doing this. Markdown is just too nice 😄

Reply:
next pr: add markdown support to news items :slightly_smiling_face: